pkg/storaged/content-views.jsx

```diff
@@ -594,7 +594,7 @@ function append_logical_volume_block(client, rows, level, block, lvol) {
             size: utils.fmt_size(block.Size),
             text: lvol.Name
         };
-        tabs = create_tabs(clienta, block, false);
+        tabs = create_tabs(client, block, false);
```

Comment:
Yes.  This code path was not executed by the tests.  It only happens for logical volumes with partition tables.  I am not sure whether it is actually possible to partition lvs.